apps/examples/wc-next13/pages/index.tsx

```diff
@@ -1,21 +1,76 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import { Inter } from 'next/font/google'
+import React, { useState, useRef } from 'react';
+import Head from 'next/head';
+import { createComponent } from '@lit-labs/react';
+import { PieButton, BUTTON_SIZE, BUTTON_VARIANT } from '@justeattakeaway/pie-button';
 
-const inter = Inter({ subsets: ['latin'] })
+const Button = createComponent({
```

Comment:
Would every React consumer have to do this the same way?

If so, could we export our components with the React wrapper already applied for React consumers?